### PR TITLE
[FIX] fixed sql and fuel detection

### PR DESCRIPTION
**Pull request by RazerGhost**
I have added back the missing comma and also fixed the way that fuel is being detected.

if the fuel string is empty it will use ox_fuel but if it is not empty it will use the defined config.FuelScript export

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -335,10 +335,10 @@ local function ParkOwnedVehicle(veh, garageName, vehLocation, plate)
 
     local totalFuel = 0
 
-    if config.FuelScript then
-        totalFuel = exports[config.FuelScript]:GetFuel(veh)
-    elseif config.FuelScript == '' then
+    if config.FuelScript == '' then
         totalFuel = Entity(veh).state.fuel
+    elseif config.FuelScript then
+        totalFuel = exports[config.FuelScript]:GetFuel(veh)
```
Comment (solareon) [inline client/main.lua]:
```suggestion
    if config.FuelScript ~= '' then
        totalFuel = exports[config.FuelScript]:GetFuel(veh)
    else
        totalFuel = Entity(veh).state.fuel
```

Little bit simpler and cleaner way to write this.

File: client/main.lua [hunk 2]
```diff
@@ -645,10 +644,11 @@ end
...
 local function UpdateVehicleSpawnerSpawnedVehicle(veh, garage, heading, vehicleConf, cb)
     local plate = GetPlate(veh)
-    if config.FuelScript then
-        exports[config.FuelScript]:SetFuel(veh, 100)
-    elseif config.FuelScript == '' then
         Entity(veh).state.fuel = 100 -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
+    if config.FuelScript == '' then
+        Entity(veh).state.fuel = 100 -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
+    elseif config.FuelScript then
+        exports[config.FuelScript]:SetFuel(veh, 100)
```
Comment (solareon) [inline client/main.lua]:
```suggestion
    if config.FuelScript ~= '' then
        exports[config.FuelScript]:SetFuel(veh, 100)
    else
        Entity(veh).state.fuel = 100 -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
```

Same here as well.

File: client/main.lua [hunk 3]
```diff
@@ -689,21 +689,21 @@ function UpdateSpawnedVehicle(spawnedVehicle, vehicleInfo, heading, garage)
             OutsideVehicles[plate] = spawnedVehicle
             TriggerServerEvent('qb-garages:server:UpdateOutsideVehicles', OutsideVehicles)
         end
-        if config.FuelScript then
-            exports[config.FuelScript]:SetFuel(spawnedVehicle, 100)
-        elseif config.FuelScript == '' then
+        if config.FuelScript == '' then
             Entity(spawnedVehicle).state.fuel = 100 -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
+        elseif config.FuelScript then
+            exports[config.FuelScript]:SetFuel(spawnedVehicle, 100)
```
Comment (solareon) [inline client/main.lua]:
```suggestion
        if config.FuelScript ~= '' then
            exports[config.FuelScript]:SetFuel(spawnedVehicle, 100)
        else
            Entity(spawnedVehicle).state.fuel = 100 -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
```

File: client/main.lua [hunk 4]
```diff
@@ -689,21 +689,21 @@ function UpdateSpawnedVehicle(spawnedVehicle, vehicleInfo, heading, garage)
...
             OutsideVehicles[plate] = spawnedVehicle
             TriggerServerEvent('qb-garages:server:UpdateOutsideVehicles', OutsideVehicles)
         end
-        if config.FuelScript then
-            exports[config.FuelScript]:SetFuel(spawnedVehicle, vehicleInfo.fuel)
-        elseif config.FuelScript == '' then
+        if config.FuelScript == '' then
             Entity(spawnedVehicle).state.fuel = vehicleInfo.fuel -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
+        elseif config.FuelScript then
+            exports[config.FuelScript]:SetFuel(spawnedVehicle, vehicleInfo.fuel)
```
Comment (solareon) [inline client/main.lua]:
```suggestion
        if config.FuelScript ~= '' then
            exports[config.FuelScript]:SetFuel(spawnedVehicle, vehicleInfo.fuel)
        else
            Entity(spawnedVehicle).state.fuel = vehicleInfo.fuel -- Don't change this. Change it in the  Defaults to ox fuel if not set in the config
```

**Review by solareon — COMMENTED** [review]
Just some minor style tweaks.

**Comment by xViperAG** [comment]
@RazerGhost If you could make these changes, that would be much appreciated. I'll pull once these are done.

**Comment by RazerGhost** [comment]
They should be fixed now with @solareon 's suggestions

**Pull request merged**